src/client/routes/saksbilde/logg/LoggProvider.tsx

```diff
@@ -21,25 +21,23 @@ interface LoggProviderProps {
 }
 
 export const LoggProvider = ({ children }: LoggProviderProps) => {
-    const aktivVedtaksperiode = useAktivVedtaksperiode() as Vedtaksperiode | UfullstendigVedtaksperiode;
+    const aktivPeriode = useAktivPeriode();
+    const vedtaksperiode = useVedtaksperiode(aktivPeriode?.id!);
```

Comment:
Kunne det ha vært noe default-opplegg her, så man slapp å bruke _to_ hooks for å få tak i perioden?